RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractEdge.java

```diff
@@ -227,13 +229,15 @@ void processWebCacheDbFile(String eseDumperPath, List<AbstractFile> webCacheFile
 
                 this.getCookies(webCacheFile, resultsDir);
 
-//                if (context.dataSourceIngestIsCancelled()) {
-//                    return;
-//                }
-//                Putting downloads on hold
-//                this.getDownload(webCacheFile, resultsDir); 
             } finally {
                 tempWebCacheFile.delete();
+                
+                // Emppty the result dir
+                File[] resultFiles = resultsDir.listFiles();
+                for (File file : resultFiles) {
+                    file.delete();
+                }
```

Comment:
You can use org.sleuthkit.autopsy.coreutils.FileUtil instead. One benefit of that is that I will soon be updating these utils to handle long paths.

By the way, there is a typo in the comment.